src/docs/app-monitoring/sysdig-monitor-setup-team.md

```diff
@@ -137,6 +148,8 @@ You should expect to see the following from the `sysdig-team` outcome:
 Message:               Awaiting next reconciliation
 Reason:                Successful
 ```
+**NOTE**: the custom resource kind is SysdigTeam, but when using the oc command, you need to use the pluralized form `sysdig-teams`. This behavior is due to the fact that OpenShift automatically generates the plural form for custom resources. So you will need to use `oc get sysdig-teams` to list all SysdigTeams
```

Comment:
It doesn't have to be pluralized though does it? From what I can see `oc get sysdig-team` performs the same as `oc get sysdig-teams`